Model/Payment_Method.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
I highly suggest that you build yourself an Payment_Method interface make an abstract class Payment_Method that implements the interface and then extend the Payment_Method class in each payment method. Allot of code is shared or same in each payment method. And in your implementation you would need to implement it over and over again.

Reply:
this is the abstract class each payment method will extend from. We'll see the shared stuff once starting implementing other payment methods.

Reply:
I give you this up front i will not accept future pull-requests of new payment methods it the code will not be reused. For now that only PayPal is implemented its ok.

Reply:
👍